vspace/src/main/webapp/WEB-INF/views/staff/modules/slides/slide.html

```diff
@@ -35,16 +35,25 @@
 			color: var(--primary) !important;
 			background: transparent;
 			}
-			
-			
+            .tooltip {
+                z-index: 0;
+            }
         </style>
         <script th:inline="javascript">
-            //# sourceURL=click.js
+            $(function () {
+		        $('.close').on('click', function () {
+		        	$('.modal').hide();
+		        })
+            })
+
+            var currentSpaceName;
             var contentCount = [[${#lists.size(slideContents)}]];
             var markDown = null;
             var defaultValue;
+            var currentSpaceId;
             var addVideoClk=0;
             var editVideoClk=0;
+            var currentBlockId;
```

Comment:
I'm am fairly certain both `currentSpaceId` and `currentBlockId` do not need to be global variables.

Reply:
without currentBlockId as a global variable close Space Block does not work as it is not able to get the id of the block, if you have any other suggestion then lmk.